File: python/src/main/python/TheUser.py
from User import *


class TheUser(User):

    def publishPermission(self):
        print("Publishing not allowed.")
        return False

    def deletePermission(self):
        print("Delete not allowed.")
        return False

    def approvePermission(self):
        print("Approve not allowed.")
        return False

    def commentPermission(self):
        print("Comment not allowed.")
        return False

File: java/src/main/java/mx/iteso/TheUser.java
package mx.iteso;

/**
 * Class TheUser.
 */
public class TheUser extends User {
    /**
     * Publish permission.
     *
     * @return boolean.
     */
    public boolean publishPermission() {
        System.out.println("Publish not allowed.");
        return false;
    }

    /**
     * Delete Permission.
     *
     * @return boolean.
     */
    public boolean deletePermission() {
        System.out.println("Delete not allowed.");
        return false;
    }

    /**
     * Approve Permission.
     *
     * @return boolean.
     */
    public boolean approvePermission() {
        System.out.println("Approve not allowed.");
        return false;
    }

    /**
     * Comment Permission.
     *
     * @return boolean.
     */
    public boolean commentPermission() {
        System.out.println("Comment not allowed.");
        return false;
    }
}
